### ISSUE TITLE
Render custom path in VictoryLine for shadow effect

### ISSUE THREAD

**Issue by sja1n** [FormidableLabs/victory-native#286]
Hello, I am trying to plot line with shadow effect something like 
[Image Reference from react-native-svg-charts](https://ibb.co/bvWVUn), I referred the [code](https://github.com/JesperLekland/react-native-svg-charts-examples/blob/master/storybook/stories/line-chart/with-shadow.js) in the example and tried to implement the same for by using `path` but in this author is getting the line which he is passing to the path which then will get plotted along the line.
I wanted to ask if something like this is possible or some workaround, like if i plot 2 line charts one with stroke with opacity of 0.2 ?

**Comment by boygirl** [comment]
@sja1n yes. I think the easiest way to do this would be to plot two lines in a single chart
